user/server-grpc-example.md

```diff
@@ -0,0 +1,156 @@
+# Example usage of the gRPC interface
+
+On the [previous section](server-protocol.md) we checked
```

Comment:
Works, because its in the same directory and they are relatives (to add the directory name it would have to be `../user/server-protocol.md`).